### PR TITLE
feat: add custom initialization messages

### PR DESCRIPTION
**Pull request by srmaxmazur**
- update message error for ErrorGetAllTags and ErrorInterateAllTags
- add custom message to flareact and javascript

### CODE REVIEW

**Review by pauloSF0 — APPROVED** [review]

File: pkg/cmd/webapp/init/init.go [hunk 1]
```diff
@@ -161,32 +161,81 @@ func (cmd *InitCmd) run(info *InitInfo, options *contracts.AzionApplicationOptio
...
 		}
 
-		if err = UpdateScript(info, cmd, path); err != nil {
+		bytePackageJson, pathPackageJson, err := ReadPackageJson(cmd, path)
+		if err != nil {
+			return err
+		}
+
+		projectName, projectSettings, err = DetectedProjectJS(bytePackageJson)
+		fmt.Fprintf(cmd.Io.Out, "Auto-detected Project Settings (%s)\n", projectSettings)
```
Comment (PatrickMenoti) [inline pkg/cmd/webapp/init/init.go:176]: Put this message in the messages package

File: Makefile [hunk 2]
```diff
@@ -89,6 +89,12 @@ build: ## build application
 	@ $(GO) version
 	@ source $(ENVFILE) && $(GO) build -ldflags "$(LDFLAGS)" -o ./bin/$(NAME) ./cmd/$(BIN)
 
+.PHONY : builddev
```
Comment (PatrickMenoti) [inline Makefile:92]: I don't see the point of creating this 🤔

File: pkg/cmd/webapp/init/init.go [hunk 3]
```diff
@@ -161,32 +161,81 @@ func (cmd *InitCmd) run(info *InitInfo, options *contracts.AzionApplicationOptio
...
 			return err
 		}
 
-		if err = UpdateScript(info, cmd, path); err != nil {
+		bytePackageJson, pathPackageJson, err := ReadPackageJson(cmd, path)
+		if err != nil {
+			return err
+		}
+
+		projectName, projectSettings, err = DetectedProjectJS(bytePackageJson)
```
Comment (PatrickMenoti) [inline pkg/cmd/webapp/init/init.go:175]: if we don't detect the framework, we should check the flag --type to see if the user has informed any.

File: Makefile (hunk 2 above)
Comment (srmaxmazur) [reply Makefile:92]: it is to facilitate my development

**Review by PatrickMenoti — APPROVED** [review]

**Pull request merged**